manifesto/personas.md

```diff
@@ -0,0 +1,63 @@
+# Kyma Personas
+
+The following personas are fictional characters, which we defined in order to represent the different user types that might use Kyma. Keep in mind those personas when contributing to Kyma as they will help you to understand Kyma users' needs and goals (GAINS :green_heart:) and major pain points (PAINS :broken_heart:).
+
+1. [Matthew](#matthew---cloud-native-software-engineer)
+2. [Nicolas](#nicolas---tech-lead)
+
+## Matthew - Cloud Native Software Engineer
```

Comment:
```suggestion
## Matthew - Cloud-Native Software Engineer
```

Reply:
As discussed, let's go with `Cloud Native` for now.